src/Microsoft.PowerApps.TestEngine/TestEngineEventHandler.cs

```diff
@@ -12,16 +12,20 @@ namespace Microsoft.PowerApps.TestEngine
     /// </summary>
     public class TestEngineEventHandler : ITestEngineEvents
     {
-        public int _casesTotal = 0;
-        public int _casesPassed = 0;
+        private int _casesTotal = 0;
+        private int _casesPassed = 0;
+
+        public int CasesPassed { get => _casesPassed; set => _casesPassed = value; }
+        public int CasesTotal { get => _casesTotal; set => _casesTotal = value; }
 
         public TestEngineEventHandler()
         {
         }
 
-        public void SetNumberOfTotalCases(int numCases)
+        public void SetAndInitializeCounters(int numCases)
```

Comment:
This is a functionality for **console output only**. And I believe by design it might be better within this one. Keep in mind these counters are specific to **TEEvent handler** and not the actually test state. 
TestState should not contain number of test cases passed or total test cases anyways since it is more TestState  is more a config file.

Reply:
Yep agreed. I think we can defer my suggestion to a separate improvement item to retrieve these data points closer to the source of truth than computing them at the console output layer - as this suggestion is not specific to this fix. The approach to this fix itself looks good to me.